src/plugin/plugin/plugin-server.js

```diff
@@ -9,12 +9,12 @@ export default class Plugin {
     if (!this.hasOwnProperty('__instances__')) {
       this.__instances__ = new WeakMap();
     }
-    if (ctx == null) ctx = __global_key__;
+    const key = ctx ? ctx : __global_key__;
```

Comment:
small nit: probably should be `ctx != null` to avoid truthy check and weird continuity behavior (e.g. throws on 1, but not on 0)